packages/devtools_app/lib/src/shared/config_specific/copy_to_clipboard/_copy_to_clipboard_web.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+// ignore: avoid_web_libraries_in_flutter, as designed
+import 'dart:html';
+
+void copyToClipboardVSCode(String data) {
+  // This post message is only relevant in VSCode. If the parent frame is
+  // listening for this command, then it will attempt to copy the contents
+  // to the clipboard in the context of the parent frame.
+  window.parent?.postMessage(
```

Comment:
You can probably use the existing `postMessage` in packages\devtools_app\lib\src\shared\config_specific\post_message\post_message.dart to avoid needing multiple stub files for this API? (eg. have a single copy function that just calls the shared postMessage)?

Reply:
interesting, yeah it doesn't look like it exposed 🤔 
Would you be alright with me just leaving the multi stub files for the copy behaviour?

Reply:
Issue created :)
https://github.com/flutter/devtools/issues/6613